buildenv/jenkins/README.md

```diff
@@ -41,10 +41,13 @@ This folder contains Jenkins pipeline scripts that are used in the OpenJ9 Jenkin
 - Request a Sanity build on zLinux all java versions
```

Comment:
We should also change line 29:

>  - Current supported PR builds are "Compile", "Compile & Sanity", or "Compile & Extended"

Reply:
That statement is still true. I think we need another sentence that specifically calls out this feature.